src/role-access/containers/exclusion/exclusion-navigation/exclusion-navigation.component.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-back-link {
+  margin-bottom: 1px;
+}
+
+.govuk-grid-column-full {
```

Comment:
Here as well we could use the GDS helper classes `govuk-!-padding-left-0`

Reply:
this is not only adding a padding left, this is telling the app to inherit all properties from .govuk-grid-column-full and override(customize) thepadding left for our scenario.